app/models/income.rb

```diff
@@ -40,7 +40,7 @@ def income_payments
                           not_working_owners, employed_income_payment_types)
     end
 
-    @income_payments = scope
+    @income_payments = scope.select(&:complete?)
```

Comment:
Not a big fan of calling an array method on what is otherwise a query different of scopes. This would prevent further scope chaining if we wanted it. Happy to hear other ideas!